src/drivers/uavcan/uavcan_params.c

```diff
@@ -335,3 +335,15 @@ PARAM_DEFINE_INT32(UAVCAN_SUB_MAG, 1);
  * @group UAVCAN
  */
 PARAM_DEFINE_INT32(UAVCAN_SUB_RNG, 0);
+
+/**
+ * subscription safety button
+ *
+ * Enable UAVCAN Safety Button.
+ *  ardupilot::indication::Button
+ *
+ * @boolean
+ * @reboot_required true
+ * @group UAVCAN
+ */
+PARAM_DEFINE_INT32(UAVCAN_SAFETY, 0);
```

Comment:
```suggestion
PARAM_DEFINE_INT32(UAVCAN_SUB_SAFETY, 0);
```

The convention I'm gradually working towards is everything being configurable via a `UAVCAN_SUB_*` or `UAVCAN_PUB_*` parameter.